services/accounts/settings.go

```diff
@@ -16,6 +16,11 @@ type SettingsAPI struct {
 }
 
 func (api *SettingsAPI) SaveSetting(ctx context.Context, typ string, val interface{}) error {
+	// NOTE(Ferossgp): Backward compatibility, skip this for older clients instead of returning error
```

Comment:
Maybe worth adding the VERSION in the comment, so we know when it's safe to remove. Very neat!